index.html

```diff
@@ -0,0 +1,300 @@
+                    <a href="#">
+                        <img src="./images/Linkedin icon.png" alt="Image missing">
+                    </a>
+                </li>
+                <li class="social-icon">
+                    <a href="#">
+                        <img src="./images/Vector.png" alt="Image missing">
+                    </a>
+                </li>
+                <li class="social-icon">
+                    <a href="#">
+                        <img src="./images/Vector.svg" alt="Image missing">
+                    </a>
+                </li>
+            </ul>
+        </section>
+
+
+
+        <!-- card-info -->
+        <section class="card-info">
+            <div class="card-item">
+                <div class="images-tags">
+                    <a href="#">
+                        <img class="img1" alt="Image missing">
+                    </a>
+                </div>
+                <div class="main-text">
+                    <h1 class="title">Tonic</h1>
+                    <ul class="frame-2">
+                        <li class="top-title">Canopy</li>
+                        <li class="details"><img src="./images/Counter.png" alt="Image missing"></li>
+                        <li class="details">Back End Dev</li>
+                        <li class="details"><img src="./images/Counter.png" alt="Image missing"></li>
+                        <li class="details">2015</li>
+                    </ul>
+                    <p class="description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="tags">
+                        <li><a href="#"><img src="./images/Tag.png" alt="Image missing"></a></li>
+                        <li><a href="#"><img src="./images/Tag (1).png" alt="Image missing"></a></li>
+                        <li><a href="#"><img src="./images/Tag(2).png" alt="Image missing"></a></li>
+                    </ul>
+                    <a href="#" class="more-button">See Project</a>
+                </div>
+            </div>
+            <div class="card-item">
+                <div class="images-tags">
+                    <a href="#">
+                        <img class="img2" alt="Image missing">
+                    </a>
+                </div>
+                <div class="main-text">
+                    <h1 class="title">Multi-Post
+                        Stories</h1>
+                    <ul class="frame-2">
+                        <li class="top-title top-title2">Canopy</li>
+                        <li class="details"><img src="./images/Counter.png" alt="Image missing"></li>
+                        <li class="details"><span class="stack">Back End Dev</span></li>
+                        <li class="details"><img src="./images/Counter.png" alt="Image missing"></li>
+                        <li class="details">2015</li>
+                    </ul>
```

Comment:
- [ ] While you did a wonderful job styling your cards, I think it would be better if you align to center each of these elements

**Your design**

![Screenshot from 2023-01-19 00-57-46](https://user-images.githubusercontent.com/105117832/213367090-87c92e95-790b-485e-861a-a3373464ffa4.png)

**Figma design**

![Screenshot from 2023-01-19 01-00-37](https://user-images.githubusercontent.com/105117832/213367235-f30726fe-c976-4f89-aad8-a6036394008c.png)